### ISSUE TITLE
[Adaptive Card iOS - Action.Execute.IsEnabled.json]: Voice over does convey the incorrect information as 'Dimmed' for non - disabled control.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8401]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.1.8.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 16.3.1
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.5" control and select it.
5. Navigate to 'Elements' control and activate it.
6. Navigate to 'Action.Execute.IsEnabled.json' and activate it.
7. Navigate to the 'Disabled' button and observe the issue.

**Actual Result:**
Voice over does convey the incorrect information as 'Dimmed' for non - disabled control i.e. Disabled button. 

**Expected Result:**
Voice over should not convey the incorrect information as Dimmed for the non-disabled 'disable' button. It should be either disabled as no action is getting performed on activating it or it should be in dimmed state.

**Note:**
1. Same issue is repro under 'Action.OpenUrl.IsEnabled.json' screen.
2. Same issue is repro under 'Action.ShowCard.IsEnabled.json' screen.
3. Same issue is repro under 'Action.Submit.IsEnabled.json' screen.

**User Experience:**
It will impact the screen reader user if the screen reader does convey the incorrect information for the Disabled button.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/227787459-65013cc9-f60b-4fc3-81ec-983cb7b90a56.MP4

### Card JSON

```JSON
NA
```
